### PR TITLE
Edits to JOSS tidyverse paper

### PR DESCRIPTION
**Pull request by batpigandme**

### CODE REVIEW

File: vignettes/paper.md [hunk 1]
```diff
@@ -107,15 +107,17 @@ bibliography: paper.bib
 
 ![](tidyverse-logo.png)
 
-At a high level, the tidyverse is a language for solving data science challenges with R code. Its primary goal is to facilitate a conversation between a human and a computer about data. Less abstractly, the tidyverse is a collection of R packages that share a high-level design philosophy and low-level grammar and data structures, so that learning one package makes it easier to learn the next. 
+At a high level, the tidyverse is a language for solving data science challenges with R code [@R-core]. Its primary goal is to facilitate a conversation between a human and a computer about data. Less abstractly, the tidyverse is a collection of R packages that share a high-level design philosophy and low-level grammar and data structures, so that learning one package makes it easier to learn the next.
```
Comment (hadley) [inline vignettes/paper.md]: I think you could now move this citation down to related work

File: vignettes/review-jeffreyhanson.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+Done!
+
+> This is a bit nitpicky but I do not think that data are necessarily "stored" in an API? I would say that data are accessed through an API? Apologies if I'm just plain wrong.
+
+We've tried to work around this by saying "behind a web API".
+
+> My impression is that terms like "data frame" and "flat file" will be very useful for individuals with experience in programming _R_, _Python_, or _SQL_. If such individuals are the intended audience for this manuscript, then using such terminology is ideal. If a broader audience is intended, then perhaps using different terms (or clarifying these terms) will be useful (e.g. terms like "tabular" or "spreadsheet" may be more informative for a broader audience).
+
+We agree that "tabular" is a bit than "flat" so we've changed this.
```
Comment (batpigandme) [inline vignettes/review-jeffreyhanson.md]:
```suggestion
We agree that "tabular" is a bit than "flat," so we've changed this.
```

File: vignettes/review-jeffreyhanson.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+We've added this, as suggested.
+
+> Is the ordering of these packages significant? If not, perhaps these packages could be listed alphabetically?
+
+Packages are now listed in alphabetical order.
+
+> Perhaps "application programming interface (API)" instead of "API" for clarity?
+
+Done!
```
Comment (batpigandme) [inline vignettes/review-jeffreyhanson.md]: Need to change this, given we went back to API…

File: vignettes/review-jeffreyhanson.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+We left this as is, since they're already in a parenthetical, and worry that "comma-separated values" might be more obscure than "csv."
+
+> I don't want to wade into the debate regarding if data is a singular or plural term, but I think that it might be useful to be consistent throughout the manuscript. My interpretation of the sentence containing the phrase "[...] import data stored in other common formats or retrieve it directly [...]" uses the term "data" to refer to both a plural and a singular entity. Perhaps the sentence might be clearer if written as something like "[...] import data from other common formats or the web directly"?
+
+Now changed to "make it possible to import data stored in other common formats or directly from the web."
+
+> The manuscript contains many embedded links which mean that URLs are not shown in the text (e.g. [like this](https://google.com)). I am not familiar enough with the correct reference style for this journal, so I thought I would raise this point just in case.
+
+Also well as links to individual websites, all packages also have full citations.
```
Comment (batpigandme) [inline vignettes/review-jeffreyhanson.md]:
```suggestion
As well as links to individual websites, all packages also have full citations.
```

File: vignettes/review-jeffreyhanson.md (hunk 3 above)
Comment (batpigandme) [inline vignettes/review-jeffreyhanson.md]:
```suggestion
We don't believe that spelling API out makes it any easier to understand.
```

**Pull request merged**